### PR TITLE
uefi: Validate user provided mount points too

### PR DESCRIPTION
**Pull request by superm1**
One more idea that can cover the straws drawn in #597

### CODE REVIEW

File: plugins/uefi/fu-uefi-common.c [hunk 1]
```diff
@@ -273,9 +273,9 @@ fu_uefi_read_file_as_uint64 (const gchar *path, const gchar *attr_name)
 }
 
 gchar *
-fu_uefi_guess_esp_path (void)
+fu_uefi_validate_esp_path (const gchar *extra_path)
```
Comment (hughsie) [inline plugins/uefi/fu-uefi-common.c]: I think if we're validating, we should be returning a GError on failure.

Comment (superm1) [reply plugins/uefi/fu-uefi-common.c]: Sounds good, I'll adjust.

Comment (superm1) [reply plugins/uefi/fu-uefi-common.c]: Adjusted

**Comment by superm1** [comment]
This one look OK now?

**Comment by hughsie** [comment]
I had a quick look, and I wasn't sure about the flow -- I'll re-review tonight, sorry for the delay.

**Comment by superm1** [comment]
Closing in preference to #629

**Pull request closed without merging**